### ISSUE TITLE
fully qualified keywords with anonymous response models

### ISSUE THREAD

**Issue by ikitommi** [metosin/compojure-api#101]
```json
{
  "my.project.domain/random": 0
}
```

where it should be:

```json
{
  "random": 0
}
```

**Comment by BinarySplit** [comment]
I found this as well when upgrading from Compojure.API 0.19.3 to 0.20.2. It only occurs on :body-params, e.g.

```clojure
(POST* "/find-pizza-by-id" []
  :body-params [id :- Long]
  (ok (get-pizza id)))
```

results in the body model:

    Body23872 {
        pizza/id (integer)
    }

The easiest workaround is to define the model explicitly and use :body instead of :body-params. For the example above, this would look like:

```clojure
(s/defschema FindPizzaByIdModel {:id Long})
...
(POST* "/find-pizza-by-id" []
  :body [{:keys [id]} GetPizzaByIdModel]
  (ok (get-pizza id)))
```

and gives the expected non-namespaced model:

    GetPizzaByIdModel {
        id (integer)
    }

**Comment by ikitommi** [comment]
thanks. we'll try to resolve this and push out new release this week.

**Comment by ikitommi** [comment]
should be fixed in `[metosin/compojure-api "0.20.2-SNAPSHOT"]`.

**Comment by ikitommi** [comment]
```clojure
[metosin/compojure-api "0.20.3"]
```
